src/OpenFeature.Contrib.Providers.Flipt/version.txt

```diff
@@ -0,0 +1 @@
+0.0.1
```

Comment:
Also add an entry in the [release-plz-manifest](https://github.com/open-feature/dotnet-sdk-contrib/blob/main/.release-please-manifest.json) with a version number matching this, or it will release the first version as 1.0.0.